atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -201,6 +190,59 @@ def add_as_an_alias(new, old, is_old_inherited):
             self.log.debug("applied only some aliases, following old labels were not found: %s",
                            ", ".join(not_applied))
 
+        def check_if_all_same(labels_to_check, source):
+            if labels_to_check:
+                label_value = source[next(iter(labels_to_check))]
```

Comment:
Why use this instead of `label_value = labels_to_check[0]` ? Both seem to work the same way for empty iterablels:

```python
>>> l = []
>>> l[0]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
list index out of range
>>> next(iter(l))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
StopIteration
```